lms/djangoapps/program_enrollments/api/v1/views.py

```diff
@@ -513,15 +551,6 @@ class ProgramCourseRunSpecificViewMixin(ProgramSpecificViewMixin):
     A mixin for views that operate on or within a specific course run in a program
     """
 
-    def check_course_existence_and_membership(self):
-        """
-        Attempting to look up the course and program will trigger 404 responses if:
-        - The program does not exist
-        - The course run (course_key) does not exist
-        - The course run is not part of the program
-        """
-        self.course_run  # pylint: disable=pointless-statement
```

Comment:
Please delete the `course_run` function too, as it's incorrect and now also unused.

Reply:
huh, you're right. I didn't realize that this was only ever called to validate that the course_run existed